raddb/sites-available/coa-relay

```diff
@@ -256,20 +238,14 @@ server coa {
 		#  Report what we did
 		#
 		if (&control.Tmp-Integer-0) {
-			update reply {
-				&Reply-Message += "Sent updates for %{control.Tmp-Integer-0} active sessions"
-			}
+			&reply.Reply-Message += "Sent updates for %{control.Tmp-Integer-0} active sessions"
 			ok
 		} else {
-			update reply {
-				&Reply-Message += "No active sessions found"
-			}
+			&reply.Reply-Message += "No active sessions found"
```

Comment:
this is string append here, too.  Not list append.

And like some others later on.